### ISSUE TITLE
Add a SemanticsBinding

### ISSUE THREAD

**Issue by Hixie** [flutter/flutter#12852]
At some point (doesn't have to be any time soon) we should add a binding to the semantics layer and move the `window.onSemanticsEnabledChanged` hook to there (from the rendering layer), in the same way that rendering handles `onTextScaleFactorChanged` (and widgets' binding then hooks it), 
since we now have a separate semantics layer.

**Comment by pedromassangocode** [comment]
I found a `mixin` called `SemanticsBinding` in the link bellow not sure if this is what is being talked here. Maybe this was already fixed?!

https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/semantics/binding.dart

**Comment by goderbauer** [comment]
This was done some time ago.
